main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = ["George Klooney",
+              "Kianu Reaves",
```

Comment:
The name 'Kianu Reaves' is misspelled. It should be 'Keanu Reeves'. This is corrected later, but it would be more efficient to use the correct name initially.